tests/parametric/test_parametric_workflow.py

```diff
@@ -0,0 +1,154 @@
+from pathlib import Path
+import tempfile
+import time
+
+import pytest
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+
+@pytest.mark.fluent_232
```

Comment:
not working in 24.1 ATM - https://github.com/ansys/pyfluent-parametric/issues/213